bundles/org.palladiosimulator.dataflow.confidentiality.analysis/src/org/palladiosimulator/dataflow/confidentiality/analysis/sequence/entity/pcm/DatabaseActionSequenceElement.java

```diff
@@ -14,7 +15,7 @@
 import org.palladiosimulator.pcm.core.composition.AssemblyContext;
 
 public class DatabaseActionSequenceElement<T extends OperationalDataStoreComponent> extends AbstractPCMActionSequenceElement<T> {
-	private final Logger logger = Logger.getLogger(DatabaseActionSequenceElement.class);
+	private final Logger logger = Logger.getRootLogger();
```

Comment:
I'm not sure whether its a good idea to always use the root logger. I get the distinction between the global log levels and the analysis log level but is the difference in console output big enough to justify the added complexity of classes receiving other loggers than the one for their own class?

Reply:
I agree. After reading some more into the documentation, it seems that using a logger for each class still uses the Logger Level set in the Root Logger. Because of that, i have exchanged  every reference to the root logger to a logger for the class